### PR TITLE
feature(cloudformation): add i3.metal to the supported list

### LINKED ISSUE #31: Missing i3.metal in  Launch a Scylla Cluster using Cloud Formation.
@tzach  Please add i3.metal instance type to Launch a Scylla Cluster using Cloud Formation.
Thanks.

### PR DESCRIPTION
**Pull request by fruch**
Guy.C asked for it, Guy got it

Fixes #31

### CODE REVIEW

File: aws/cloudformation/scylla.yaml [hunk 1]
```diff
@@ -151,6 +152,7 @@ Resources:
             - '"}]}]'
             - !Sub ', "cluster_name": "${ClusterName}", '
             - '"endpoint_snitch": "org.apache.cassandra.locator.Ec2Snitch"}, '
+            - '"start_scylla_on_first_boot": true, '
```
Comment (bentsi) [inline aws/cloudformation/scylla.yaml:155]: how is this related to i3.metal?

Comment (fruch) [reply aws/cloudformation/scylla.yaml:155]: It's not
but I've change the jinja2 template, and seem like we forgot to put this change from it.

I've tested it out, and it's working as expected

Comment (fruch) [reply aws/cloudformation/scylla.yaml:155]: I've separated it into it's own commit

Comment (bentsi) [reply aws/cloudformation/scylla.yaml:155]: thanks
